### DISCUSSION TITLE
Doji family of candlestick patterns

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#734] [Indicators]
Community discussion about the Doji family of candlestick patterns

- [Doji](https://dotnet.stockindicators.dev/indicators/Doji/#content)
- Morning star
- Evening star
- Long-legged
- Gravestone
- Dragonfly
- Takuri

**Comment by phivanloi** [comment]
Some one convert from here. 
https://github.com/QuantConnect/Lean/tree/master/Indicators/CandlestickPatterns
